src/admin-dhis2/admin-dhis2.service.js

```diff
@@ -81,9 +81,11 @@
                 url: openlmisUrlFactory('/api/serverConfiguration/:serverId/datasets/:datasetId/elementsAndCombos'),
                 method: 'GET'
             },
-            syncServers: {
-                url: openlmisUrlFactory('/api/execute'),
-                method: 'POST'
+            syncServer: function(serverId, datasetId) {
+                return {
+                    url: openlmisUrlFactory('/api/execute?serverId=' + serverId + '&datasetId=' + datasetId),
```

Comment:
are theses parameters required ? I'm wondering how this code will work if one of this parameter will be null

Reply:
these parameters are required, none of them can be null